osc/commandline.py

```diff
@@ -8675,7 +8675,20 @@ def do_vc(self, subcmd, opts, *args):
         else:
             apiurl = self.get_api_url()
 
-        # set user's email if the mailaddr env variable is not set
+        # set user's full name and email if the realname and mailaddr env variables are not set
+
+        if 'realname' in os.environ:
+            pass
+        elif 'realname' in conf.config['api_host_options'][apiurl]:
+            os.environ['realname'] = conf.config['api_host_options'][apiurl]['realname']
```

Comment:
"realname" is not a good idea for a global ENV variable, but use something like "OSC_VC_REALNAME" or alike.

Reply:
Do we also want to change the "mailaddr" one too? It's pretty generic as well...